platform-includes/getting-started-sourcemaps/javascript.nuxt.mdx

```diff
@@ -24,3 +24,12 @@ export default defineNuxtConfig({
   }
 });
 ```
+
+Nuxt sets default values for the source maps ([Nuxt docs](https://nuxt.com/docs/api/nuxt-config#sourcemap)) and the SDK currently keeps the source maps settings if they are explicitly defined.
+To enable this source maps setting, add the following code to you Nuxt config:
```

Comment:
I think as a first-time reader, I'd a bit confused here. We start out with an explanation but do I have to configure something explicitly to get source maps working?  Maybe rephrase it a bit by telling users what they should set first. Any explanations that are necessary, we can make, probably underneath the code block. wdyt?